### DISCUSSION TITLE
All the missing things in Mobile App

### DISCUSSION THREAD

**Discussion by hslee2008** [community/community#18455] [Mobile] [answered]
# Repos
1. Simple code editing
2. Fork/Delete/Rename/e.t.c
3. Search files/keywords

# Pull Requests
1. Commit requests

# Profile Page
1. Contribution Graph, Activity
4. Achievements
5. Organizations
6. Project, Packages

**Comment by ale-lit** [comment]
And search in your repositories?

https://github.com/github-community/community/discussions/18368

**Comment by hslee2008** [comment, marked as answer]
A lot has been fixed at this point.

**Comment by ascarobiondo007** [comment]
<link crossorigin="anonymous" media="all" integrity="
sha512-ksfTgQOOnE+FFXf+yNfVjKSlEckJAdufFIYGK7ZjRhWcZgzAGcmZqqArTgMLpu90FwthqcCX4ldDgKXbmVMeuQ=="
rel="stylesheet" href="
https://github.githubassets.com/assets/light-92c7d381038e.css" /><link
crossorigin="anonymous" media="all" integrity="
sha512-1KkMNn8M/al/dtzBLupRwkIOgnA9MWkm8oxS+solP87jByEvY/g4BmoxLihRogKcX1obPnf4Yp7dI0ZTWO+ljg=="
rel="stylesheet" href="
https://github.githubassets.com/assets/dark-d4a90c367f0c.css" /><link
data-color-theme="dark_dimmed" crossorigin="anonymous" media="all" integrity
="
sha512-cZa7DZqvMBwD236uzEunO/G1dvw8/QftyT2UtLWKQFEy0z0eq0R5WPwqVME+3NSZG1YaLJAaIqtU+m0zWf/6SQ=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/dark_dimmed-7196bb0d9aaf.css" /><link
data-color-theme="dark_high_contrast" crossorigin="anonymous" media="all"
integrity="
sha512-WVoKqJ4y1nLsdNH4RkRT5qrM9+n9RFe1RHSiTnQkBf5TSZkJEc9GpLpTIS7T15EQaUQBJ8BwmKvwFPVqfpTEIQ=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/dark_high_contrast-595a0aa89e32.css"
/><link data-color-theme="dark_colorblind" crossorigin="anonymous" media="
all" integrity="
sha512-XpAMBMSRZ6RTXgepS8LjKiOeNK3BilRbv8qEiA/M3m+Q4GoqxtHedOI5BAZRikCzfBL4KWYvVzYZSZ8Gp/UnUg=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/dark_colorblind-5e900c04c491.css" /><
link data-color-theme="light_colorblind" crossorigin="anonymous" media="all"
integrity="
sha512-3HF2HZ4LgEIQm77yOzoeR20CX1n2cUQlcywscqF4s+5iplolajiHV7E5ranBwkX65jN9TNciHEVSYebQ+8xxEw=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/light_colorblind-dc71761d9e0b.css" />
<link data-color-theme="light_high_contrast" crossorigin="anonymous" media="
all" integrity="
sha512-+J8j3T0kbK9/sL3zbkCfPtgYcRD4qQfRbT6xnfOrOTjvz4zhr0M7AXPuE642PpaxGhHs1t77cTtieW9hI2K6Gw=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/light_high_contrast-f89f23dd3d24.css"
/><link data-color-theme="light_tritanopia" crossorigin="anonymous" media="
all" integrity="
sha512-AQeAx5wHQAXNf0DmkvVlHYwA3f6BkxunWTI0GGaRN57GqD+H9tW8RKIKlopLS0qGaC54seFsPc601GDlqIuuHg=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/light_tritanopia-010780c79c07.css" />
<link data-color-theme="dark_tritanopia" crossorigin="anonymous" media="all"
integrity="
sha512-+u5pmgAE0T03d/yI6Ha0NWwz6Pk0W6S6WEfIt8veDVdK8NTjcMbZmQB9XUCkDlrBoAKkABva8HuGJ+SzEpV1Uw=="
rel="stylesheet" data-href="
https://github.githubassets.com/assets/dark_tritanopia-faee699a0004.css" />
<link crossorigin="anonymous" media="all" integrity="
sha512-Q6BI1w8BeqccNEqSfEMLAdK49Gi4JWqmdVesEGEPkZl6eaqnCoHFZsP8J2RegmbcD3K0p4udDlPOo2adIEf82Q=="
rel="stylesheet" href="
https://github.githubassets.com/assets/frameworks-43a048d70f01.css" />

**Comment by ChalsLapWork** [comment]
Contribution Graph, Activity. Please, i need it.
